cmake/simdutf-flags.cmake

```diff
@@ -14,6 +14,30 @@ if (NOT CMAKE_BUILD_TYPE)
   endif()
 endif()
 
+#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native")
+#set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=native")
+
+#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}  -mavx512vpopcntdq")
+#set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mavx512vpopcntdq")
+
+include(CheckCXXSourceRuns)
```

Comment:
This is fine for prototyping, but you should not change simdutf-flags.cmake in the final version.

Instead we want to edit our ISA detection to include a test...

Compare 
https://github.com/simdutf/simdutf/blob/25cbc995f1f63ae7f254b8a461fbe698b7821a95/include/simdutf/internal/isadetection.h#L209

with...


https://github.com/RoaringBitmap/CRoaring/blob/63e22ab72c16aff79da55d00d1189682fdf3a229/src/isadetection.c#L218

And then we want to change the target macro, see

https://github.com/simdutf/simdutf/blob/25cbc995f1f63ae7f254b8a461fbe698b7821a95/src/simdutf/icelake.h#L39


It is not very difficult but it can be confusing if you don't know where things are.